src/unstuck-bridge-tx.md

```diff
@@ -0,0 +1,305 @@
+![](/.gitbook/assets/unstucktx-approved-continue.png)
+
+### Step 2
+
+Find the transaction hash of your approved (but stuck) transaction.
+
+- If continuing from step 1 your RPC call only needs slight adjusting, click on “include option” but this time select “Approvals Ready”, Then Submit RPC Call
+  ![](/.gitbook/assets/unstucktx-approvals-ready.png)
+
+The result is a list of all SORA bridge transactions that have been
+approved on the SORA side. Your latest bridge transaction is the last
+one (scroll down if necessary), if you have made more SORA>ETH bridge
+transactions after the stuck one, you will need to go from the bottom
+to the top of the list until you find your stuck transaction.
+
+- Copy your “stuck” transaction hash, it will look like this:
+
+`0x57533dc46a5d257b6dj0nhagb0f8s4dfedfl6cpppcoaidu99dd0sdc4def4g6b`
+
+![](/.gitbook/assets/unstucktx-stuck-tx.png)
+
+### Step 3
+
+Use your transaction hash to get the required data (proof) for step 4.
+
+- Continuing from step 2, click on “get**Account**Requests(accountId, statusFilter)” and change it to “get**Approved**Requests(requestHashes, networkId)
+
+![](/.gitbook/assets/unstucktx-get-approved-requests.png)
+
+- Click on “Add item”
+- Paste in your Approved transaction hash from step 2 instead of 0x000…
+- Click “Include Option” and leave the network id as 0
+- Submit the RPC Call
+
+![](/.gitbook/assets/unstucktx-add-approved-request.png)
+
+The result is your “Proof”, it contains all the data needed for step
+4, below is an **example** of a proof, you only need the data colour
+highlighted in this example, the rest can be ignored, underneath this
+example are instructions on how yo should prepare **your** data for the next step.
+
+```
+Transfer: {
+
+currency_id: {
+
+TokenAddress: 0x6b175474e89094c44da98b954eedeac495271d0f
+
+},
+
+amount: 30,000,000,000,000,000,000,000,
+
+to: 0xd7230339a4b355ae6d24a7378cc910o85l34k5nb,
+
+from: 0x6ef01fd3b42ed7a3fa0e7ca366d0a464dc71016d,
+
+tx_hash: 0x235517c869f0288ad8jjhag6f0de0jbkelf6ooi5ycct5g9d945d73d40d987y6b,
+
+network_id: 0x0000000000000000000000000000000000000000000000000000000000000000,
+
+raw: 0x6b175474e89094c44da98b954eedeac495271d0f00000000000000000000000000000000000000000000069c1d44c523e2f80000d7330339a1b395ae5d24ab376cc610085034458c6
+{
+
+r: 0x7a42sefc5dh232j34g61d22c53b28cn9600f4jhggf70fdfeh83kb4l7d9s2bcea,
+
+s: 0x79b1e77bb20663953981d267fbc01d53f9cc3x34s63aa7h3f4385l04td9be9c43,
+
+*v: 0
+
+},
+
+{
+
+r: 0xcc9fcee0fd50bfa29ekb10ldlkjh3tdef02f9beb74a1v295ac0c5x01253f8f8f,
+
+s: 0xkafvd79ffa27g10h928jb88066e8252dec3d58fb6ebf0862c6839d880f858a8f,
+
+v:
+```
+
+**You will be copying and pasting data from your proof. The first time you do this, it's easier if you first copy and paste the whole proof into a notepad, and edit the results as follows, ready to use into step 4.**
+
+- Token Address
```

Comment:
this part better works as a table, e.g. something like this:

|Field|Guidance|Example|
|--|--|--|
|Token Address|This can be copied and pasted as it is, no commas or spaces. | `0x6b175474e89094c44da98b954eedeac495271d0f`|
|Amount|Copy and paste this number and delete all the commas. This amount will reflect the amount you bridged but will be a much larger number, for example the amount in this example bridge was actually 30000. | `30000000000000000000000`|
|...|...|.....|

or maybe a set of multiple tables since for r, s, v fields we need to add an additional note beforehand. Alternatively, you can have the basic info about r, s, v as part of the bigger table but then add a section specifically about these values since there are a lot of details